c5_it-security/1_understanding-security-threads/graded-assessments/quiz-understanding-security-threats.md

```diff
@@ -20,8 +20,8 @@ A network-based attack where one attacking machine overwhelms a target with traf
 
 When cleaning up a system after a compromise, you should look closely for any ______ that may have been installed by the attacker.
 
-* Poisoned DNS caches [INCORRECT]
-* Backdoors
+* Poisoned DNS caches
+* **Backdoors**
```

Comment:
Maybe you can update the "Latest Submission Grade" after correcting, and explanation of the answer like any other question (after ">"). Thanks!